Library/Homebrew/compat/ENV/shared.rb

```diff
@@ -3,4 +3,8 @@ def j1
     odeprecated "ENV.j1", "ENV.deparallelize"
     deparallelize
   end
+
+  def java_cache
+    odeprecated "ENV.java_cache"
```

Comment:
I'd hold off adding the `odeprecated` in favour of an `audit` for now. Comment it out and we can reenable it when people have had the chance to address the audit.

Reply:
👍 Will tweak.